content/03.aim1.md

```diff
@@ -46,20 +48,79 @@ Simulation statistics under different parameters:
 
 Figure 1: Distribution of mean expression level of simulated genes across samples under different simulated parameters.
 
-#### 2. Visualization of simulated single cell data using VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE and PCA under different parameters
+##### 2. Visualization of simulated single cell data using VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE and PCA under different parameters
 
-To see if different methods can recover cell types, we compared the 2D visualizations on simulated datasets.
-2-layer VAE performs much better to differentiate different cell types when comparing with 1-layer VAE.
+To see if different methods can recover celltypes, we compared the 2D visualizations on simulated datasets.
```

Comment:
I know @gwaygenomics suggested keeping things consistent with `celltypes`, but I think "cell types" is more grammatically correct.